src/transformers/models/whisper/convert_openai_to_hf.py

```diff
@@ -102,7 +108,23 @@ def make_linear_from_emb(emb):
     return lin_layer
 
 
-def _download(url: str, root: str) -> io.BytesIO:
+def _download(url: str, target: str) -> str:
+    with urllib.request.urlopen(url) as source, open(target, "wb") as output:
+        with tqdm(
+            total=int(source.info().get("Content-Length")), ncols=80, unit="iB", unit_scale=True, unit_divisor=1024
+        ) as loop:
+            while True:
+                buffer = source.read(8192)
+                if not buffer:
+                    break
+
+                output.write(buffer)
+                loop.update(len(buffer))
+
+    return target
+
+
+def _download_model(url: str, root: str) -> Any:
```

Comment:
not sure this is required no ?

Reply:
just moved it out since there were two types of things to be downloaded; but if you think this is not necessary, then indeed it is not.

Reply:
thanks